packages/p2p-v2/src/pages/orders/components/OrderStatusTag/OrderStatusTag.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import clsx from 'clsx';
+import './OrderStatusTag.scss';
+
+type TOrderStatusTagProps = {
+    shouldHighlightAlert?: boolean;
+    shouldHighlightDanger?: boolean;
+    shouldHighlightDisabled?: boolean;
+    shouldHighlightSuccess?: boolean;
+    status: string;
+};
+
+const OrderStatusTag = ({
+    shouldHighlightAlert,
+    shouldHighlightDanger,
+    shouldHighlightDisabled,
+    shouldHighlightSuccess,
```

Comment:
do we not need to default these to false? since these are all optional

Reply:
better to default it to false rather than leave it undefined. https://github.com/binary-com/deriv-app/pull/8464#discussion_r1295404323

Reply:
yes, but i mean why not set it like this:
```suggestion
    shouldHighlightAlert = false,
    shouldHighlightDanger = false,
    shouldHighlightDisabled = false,
    shouldHighlightSuccess = false,
```